src/style.css

```
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}
```

you can add this style for all elements with *

```suggestion
* {
  box-sizing: border-box;
}
```